evento/src/components/TabContainer/index.js

```diff
@@ -6,18 +6,34 @@ const Tab = (props) => {
 	return (
 		<div className="Tab">
 			<h4>
-				<Link to={props.path}> {props.title} </Link>
+				<Link to={props.path} onClick={() => props.onClick(props.id)}> {props.title} </Link>
 			</h4>
 		</div>
 	);
 };
 
 class TabContainer extends Component {
+	constructor() {
+		super();
+		this.state = { selectedId: 1 };
```

Comment:
This is not correct. If we enter the page from url /myevents then the selectedId should be 2. And as mentioned below, no need to use id. At the very least, move this to an other method (componentWillMount?) and set the initial state to the first one in the this.props.tabs list. Or even better, check that which tab's .path field the current url matches and select from that.